OpenSim/Simulation/Test/testForces.cpp

```diff
@@ -834,11 +841,11 @@ void testExpressionBasedBushingForce()
         "ball", Vec3(0), Vec3(0), 
         transStiffness, rotStiffness, transDamping, rotDamping);
     
-    spring.setName("linear_bushing");
+    spring.setName("translational_linear_bushing");
     
     osimModel->addForce(&spring);
     
-    osimModel->print("ExpressionBasedBushingForceModel.osim");
+    osimModel->print("TranslationalExpressionBasedBushingForceModel.osim");
```

Comment:
Whoops… got some wires crossed. I was thinking that perhaps the "translational" and "rotational" prefixes should instead be suffixes so the output files are adjacent when sorted by name. Not a big deal.